managers.md

```diff
@@ -1,115 +1,107 @@
 
 ### Management & Leadership
 
-- You work with your team to establish, contribute, and drive best practices.
-- You identify areas that could be improved and partner with others on the engineering team to improve processes within engineering.
-- You are a role model and celebrate people on your team and the broader engineering team who are acting as role models themselves.
-- You set clear goals and expectations.
-- You provide continuous feedback on whether goals and expectations are being met and how engineers can improve.
-- You are focused on identifying and supporting ways to help the members on your team learn and grow through clear career development goals and objectives.
-- You coach members of your team and help them grow and develop.
-- You work with the engineering leadership team, your teams, and corresponding stakeholders to set a successful strategy and vision for your functional area.
+- You manage a single team.
+- You manage & coach individual contributors (ICs). You put ICs up for promotion as appropriate.
+- You plan for team needs, through suggesting headcount (including levels & stack), proposing tools & integrations that support team goals, and identifying dependencies.
+- You own hiring on the team. You identify gaps in skills and be responsible for building a talented and diverse portfolio on the team.
+- You own overall retention strategy on the team.
+### Software Engineering & Design
+
+- You ensure the team has strong technical best practices. You provide the team with opportunities to enhance technical skills.
+- You are a technical and interpersonal role model. You help resolve technical issues and advise on technical decisions.
+
+### Execution & Results
+
+- You define processes (e.g. grooming, retro, team syncs, etc) for the team with cross-functional counterparts and the team directly.
+- You ensure all members of your team have the opportunity to demonstrate project ownership. You may opt to take project management on for high stakes or high urgency initiatives.
+- You shield the team from external changes (to the degree possible). You keep the team focused on the tasks explicitly assigned. You influence the decisions that are made regarding your team directly.
+
+### Collaboration & Communication
+
+- You partner with relevant counterparts (e.g. Product, Design, Marketing, Curriculum, Data Science, etc…) to build team roadmaps, establish best practices.
+- You escalate challenges the team is facing to the larger engineering (or engineering leadership) team, and suggest solutions.
+
+### Community & Citizenship
+
+- You own the success of your entire team. You ensure the team is celebrated for successes and has opportunities to learn from failure. 
+- You start to own at least one engineering-wide initiative (e.g. tech community pillar, interviewing, revising competencies)
+
+## Senior Engineering Manager
+
+You are an experienced manager who manages up to two teams directly or
```

Comment:
Should we add something about managing 1 large team or two teams?

Reply:
Sure, that seems to be the implications of the EM constraints.  I'll make this change.